test/test_nn.py

```diff
@@ -9636,6 +9636,31 @@ def helper(n, c, h, w, kernel_size, stride=None):
         helper(10, 512, 31, 31, 3, stride=2)
         helper(1, 129, 8, 8, 3, stride=2)
 
+    @onlyCUDA
+    def test_max_pool2d_indices(self, device):
+        def helper(n, c, h, w, ks):
+            if h is None:
+                x = torch.randn(n, c, w, device='cuda', dtype=torch.float, requires_grad=True)
```

Comment:
the test (while correct) is confusing - 3d input is chw, not ncw (that it, it lacks batch dimension, not one of the spatial dimensions).

Reply:
Oops, seems like I confused that with the unsqueeze(2) in max_pool1d. Will fix. Thanks!